cli/src/lib/npm/npmProjectUtils.js

```diff
@@ -74,6 +78,51 @@ export async function findPackageJsonPath(pathStr: string): Promise<string> {
   return path.join(pkgJsonPathStr, 'package.json');
 }
 
+export async function findWorkspacesPackagePaths(
+  pkgJson: PkgJson,
+  cwd: string,
+): Promise<string[]> {
+  if (
+    pkgJson.content.private !== true ||
+    !Array.isArray(pkgJson.content.workspaces)
```

Comment:
It seems that yarn has updated their api over time, but my projects seem to use `workspaces.packages` can you make it support that also so that flow-typed has the best compatibility with the end user?

Reply:
Sorry, I don't really understand your question. 

`workspaces` field seems to be `string[]` according to:
- npm: https://docs.npmjs.com/cli/v7/using-npm/workspaces
- yarn: https://classic.yarnpkg.com/lang/en/docs/workspaces

Do you mean you have, `workspaces.packages` field? 

https://docs.npmjs.com/cli/v7/configuring-npm/package-json#workspaces

Reply:
Yeah I think docs have since updated but they still support the old way. I implemented workspaces way back when it was first released and never really touched it since, my package.json looks like this
```
  "workspaces": {
    "packages": [
      "packages/*"
    ]
  },
```
Do you think we should continue to support this so that if I want to make use of this I also don't need to change the way my project is configured?